modules/aws-generate-cert/README.md

```diff
@@ -0,0 +1,26 @@
+# AWS generate cert  
```

Comment:
You can remove the "route53 zone with cert" module we talked about - it doesn't really follow good practices of separation.